TeachEasy/src/teacheasy/data/PictureData.java

```diff
@@ -0,0 +1,60 @@
+package teacheasy.data;
+
+import teacheasy.data.PageObject.PageObjectType;
+
+/**
+ * A class to hold Picture Data
+ * @author sh1157 sr896
+ *
+ */
+public class PictureData extends PageObject {
+	String url;
+	String uri;
+	float xScaleFactor, yScaleFactor, rotation;

```

Comment:
Style: Coding standard recommends one variable declaration per line. It seems reasonable to group variables with related purposes (x and y) but perhaps better to have a new line for rotation?